lib/couchbase-orm/proxies/collection_proxy.rb

```diff
@@ -0,0 +1,53 @@
+require "couchbase"
+
+module CouchbaseOrm
+    class CollectionProxy
+
+        def get!(id, **options)
+            @proxyfied.get(id, Couchbase::Options::Get.new(**options))
+        end
+
+        def get(id, **options)
+            @proxyfied.get(id, Couchbase::Options::Get.new(**options))
+        rescue Couchbase::Error::DocumentNotFound
+            nil
+        end
+
+        def get_multi!(*ids, **options)
+            result = @proxyfied.get_multi(*ids, Couchbase::Options::GetMulti.new(**options))
+            first_result_with_error = result.find(&:error)
+            raise first_result_with_error.error if first_result_with_error
+            result
+        end
+
+        def get_multi(*ids, **options)
+            result = @proxyfied.get_multi(*ids, Couchbase::Options::GetMulti.new(**options))
+            result.reject(&:error)
+        end
+
+        def remove!(id, **options)
+            @proxyfied.remove(id, Couchbase::Options::Remove.new(**options))
```

Comment:
we should raise custom couchbase orm error here

Reply:
nice to have https://github.com/Mapotempo/couchbase-orm/issues/41